docs/src/pages.js

```diff
@@ -47,18 +50,21 @@ datagrid.children =
         },
         { pathname: '/components/data-grid/demo' },
         { pathname: '/components/data-grid/getting-started' },
+        { pathname: '/components/data-grid/layout' },
         { pathname: '/components/data-grid/columns' },
         { pathname: '/components/data-grid/rows' },
         { pathname: '/components/data-grid/sorting' },
         { pathname: '/components/data-grid/filtering' },
         { pathname: '/components/data-grid/pagination' },
         { pathname: '/components/data-grid/selection' },
         { pathname: '/components/data-grid/editing', title: '🚧 Editing' },
-        { pathname: '/components/data-grid/rendering' },
-        { pathname: '/components/data-grid/export', title: 'Export & Import' },
-        { pathname: '/components/data-grid/localization', title: 'Localization' },
-        { pathname: '/components/data-grid/group-pivot', title: '🚧 Group & Pivot' },
+        { pathname: '/components/data-grid/export' },
```

Comment:
I guess you already know but we need to make the same change in the core. I'm adding it here so that we don't forget.

Reply:
@DanailH hold on, it's based on #1375. I don't think that we should or even can review it until it's merged.